deployment/stacks/api.ts

```diff
@@ -0,0 +1,402 @@
+    ),
+  };
+
+  const additionalPolicies = [
+    new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      actions: [
+        "dynamodb:DescribeTable",
+        "dynamodb:Query",
+        "dynamodb:Scan",
+        "dynamodb:GetItem",
+        "dynamodb:PutItem",
+        "dynamodb:UpdateItem",
+        "dynamodb:DeleteItem",
+      ],
+      resources: [
+        ...tables.map((table) => table.arn),
+        ...tables.map((table) => `${table.arn}/index/*`),
+      ],
+    }),
+    new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      actions: ["s3:GetObject", "s3:ListBucket", "s3:PutObject"],
+      resources: [
+        `${wpFormBucket.bucketArn}/formTemplates/*`,
+        wpFormBucket.bucketArn,
+        `${wpFormBucket.bucketArn}/formTemplates/*`,
+        `${wpFormBucket.bucketArn}/fieldData/*`,
+        sarFormBucket.bucketArn,
+        `${sarFormBucket.bucketArn}/formTemplates/*`,
+        `${sarFormBucket.bucketArn}/fieldData/*`,
+      ],
+    }),
+    new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      actions: [
+        "dynamodb:DescribeStream",
+        "dynamodb:GetRecords",
+        "dynamodb:GetShardIterator",
+        "dynamodb:ListShards",
+        "dynamodb:ListStreams",
+      ],
+      resources: tables.map((table) => table.streamArn).filter(isDefined),
+    }),
+    new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      actions: ["s3:GetObject", "s3:PutObject", "s3:ListBucket"],
+      resources: [
+        wpFormBucket.bucketArn,
+        sarFormBucket.bucketArn,
+        `${wpFormBucket.bucketArn}/fieldData/*`,
+        `${sarFormBucket.bucketArn}/fieldData/*`,
+      ],
+    }),
+  ];
+
+  const commonProps = {
+    brokerString,
+    stackName: `${service}-${stage}`,
+    api,
+    environment,
+    additionalPolicies,
+    iamPermissionsBoundary,
+    iamPath,
+  };
+
+  const requestValidator = new apigateway.RequestValidator(scope, `Validator`, {
+    requestValidatorName: `${commonProps.stackName} | Validate request body and querystring parameters`,
+    restApi: api,
+    validateRequestParameters: true,
+    validateRequestBody: true,
+  });
+
+  new Lambda(scope, "createBanner", {
+    entry: "services/app-api/handlers/banners/create.ts",
+    handler: "createBanner",
+    path: "/banners/{bannerId}",
+    method: "POST",
+    requestParameters: ["bannerId"],
+    requestValidator,
+    ...commonProps,
+  });
+
+  new Lambda(scope, "deleteBanner", {
+    entry: "services/app-api/handlers/banners/delete.ts",
+    handler: "deleteBanner",
+    path: "/banners/{bannerId}",
+    method: "DELETE",
+    requestParameters: ["bannerId"],
+    requestValidator,
+    ...commonProps,
+  });
+
+  new Lambda(scope, "fetchBanner", {
+    entry: "services/app-api/handlers/banners/fetch.ts",
+    handler: "fetchBanner",
+    path: "/banners/{bannerId}",
+    method: "GET",
+    requestParameters: ["bannerId"],
+    requestValidator,
+    ...commonProps,
+  });
```

Comment:
create and fetch banner here are following on old pattern. They should now both use 
```js
path: "/banners"
```
and have no request parameters

Reply:
Thanks @gmrabian .  Fixing that now.